utils/interfaces/schemas/library/debugger/v1/diagnostics-request.json

```diff
@@ -1,60 +1,73 @@
 {
   "$id": "/library/debugger/v1/diagnostics-request.json",
-  "type": "array",
-  "items": {
-    "type": "object",
-    "properties": {
-      "headers": {
-        "type": "object",
-        "patternProperties": {
-          ".*": { "type": "string" }
-        }
-      },
-      "content": {
-        "type": "array",
-        "items": {
+  "oneOf": [
```

Comment:
Just a suggestion here.

`oneOf` in JSON schema makes debugging very hard, because any error deep inside it will makes the output be : `the XX point does not respect any of schema in $`.

And good luck to know what is the actual error 😢 

The hack for this is to use this : 

```
allOf : [
  {
    if : type = object
    then : object_def
  },
  {
    if : type = array
    then : array_def
  },
  {
    type = [array, object]
  }
]
```

It's terribly verbose. but it'll make the error way more explicit as it'll go direclty on the failing bit.

Ping me if you need any help to make my terrible pseudo JSON schema valid 😅

Reply:
Fixed 👍